README.md

```diff
@@ -1,4 +1,10 @@
-# Mattermost Profanity Filter Plugin (Beta) ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-profanity-filter/master.svg)
+# Mattermost Profanity Filter Plugin (Beta)
+
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-profanity-filter/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-profanity-filter)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-profanity-filter/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-profanity-filter)
```

Comment:
This will require someone with GH admin privileges to go to 

https://codecov.io/gh/mattermost/mattermost-plugin-profanity-filter
* Settings > General > GitHub Webhook > Create new webhook

Reply:
@levb :point_up: